manifests/init.pp

```diff
@@ -31,6 +32,14 @@
     source  => 'puppet:///modules/pe_metric_curl_cron_jobs/tk_metrics'
   }
 
+  file { '/opt/puppetlabs/bin/puppet-metrics-collector':
```

Comment:
I like the idea of putting scripts intended to be executed by users in a `bin/` directory. We don't have one in the module. `/opt/puppetlabs/bin` is already a known location we put our utilities in, and contains many more utilities than the ones we link to `/usr/local/bin`. It seems to be the appropriate place for it. The possible counter-argument is for self-containment, in which case creating a `/opt/puppetlabs/pe_metric_curl_cron_jobs/bin` directory would be the equivalent.

We currently have a per-component bin dir already in our `/opt/puppetlabs/<thing>` pattern, but we symlink back everything we expect users to run to `/opt/puppetlabs/bin`.